rosdep/python.yaml

```diff
@@ -1230,15 +1254,23 @@ python-github-pip:
       packages: [PyGithub]
 python-gitlab:
   debian:
-    '*': [python-gitlab]
-    jessie:
+    '*':
       pip:
         packages: [python-gitlab]
+    buster: [python-gitlab]
+    sid: [python-gitlab]
```

Comment:
Please don;t add sid as per contributions guildelines: "However please don't target 'sid' as it's a rolling target and when the keys change our database gets out of date."